episodes/18-preparing.md

```diff
@@ -48,27 +47,57 @@ but reduces a lot the preparation time of future lessons and are super useful to
-    - minute cards
-    - one up, one down
-    - perhaps even some more specific feedback, for a relatively short trial run e.g. how clear were the exercises? were they appropriately challenging i.e. learner felt like they had to think about it to complete the challenge, but not so difficult that they gave up/became frustrated.
-  - link to relevant section of Instructor Training
+Figures and other visual aids are great tools to help participants understand the content being presented and make associations between pieces of information. If you are planning on using these in your lesson or teaching session - remember to enhance them with the ALT text (invisible description of images which are read aloud to blind users on screen readers providing alternative text-based format for images) or place explanatory text adjacent to relevant part of visualisation.
+  
+If your lesson requires participants to use any software or tools - it is worthwhile investing time now into creating clear setup and installation instructions for participants to follow and circulating them ahead of the workshop. This will give you more time at the pilot for teaching the lesson as troubleshooting installation problems on the spot is often time-consuming and stressfull. Each Carpentries lesson has a dedicated place for this purpose within The Carpentries Workbench located in the file `setup.md` in the lesson project root. 
+
+::::::::::::::::::::::::::::::::::::::  challenge
+
+## Exercise: Create Setup Instructions (10 minutes)
+
+Create `setup.md` file in your lesson project root with a list of software and tools needed by participants to follow your lesson and links on how to install them.
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+Another standard piece of The Carpentries Workbench are *Instructor Notes* pages - intended to help other instructors deliver your lesson by providing useful tips for delivering or explainng "sticky" points in the lesson. Instructor Notes for your lesson are located in `instructors/instructor-notes.md` file off your lesson project root where you may consider adding the following information:
+
+- rationale, strengths and/or weaknesses of the lesson design
+- what has been tried before and what worked/did not work
+- tips for teaching the lesson/particular sections
+- challenges encountered
+- troubleshooting common setup/installation/exercise issues 
+- parts that seem to cause trouble or most confusion for learners (i.e. where to tread carefully, spend more time explaining)
+    
+It may initially be difficult to create Instructor Notes until you have taught the lesson yourself for the first time - but after the first pilot you will have enough feedback to draft them.
+
+::::::::::::::::::::::::::::::::::::::  challenge
+
+## Exercise: Create Instructor Notes (10 minutes)
+
+Create `instructors/instructor-notes.md` file in your lesson project root with a list of delivery instructions for future instructors.
```

Comment:
```suggestion
Create the `instructors/instructor-notes.md` file in your lesson project root with a list of delivery instructions for future instructors.
```

Reply:
The file is already present, so they do not need to create it. I think it is true that it might be difficult for participants to add much to this file yet, so 10 minutes might be too long for this exercise TBH.

Reply:
Yes, i agree @tobyhodges - last night as I was mulling things over in my head I thought 5 minutes would be enough at this point.

Reply:
Fixed via 1729694.